### ISSUE TITLE
Little Info on HTTP

### ISSUE THREAD

**Issue by facekapow** [kesla/runtime-node-net#5]
So I decided that creating `net` servers every time I wanted to use `http` sucked, so I found this out. I split my files up separately, so in my `index.js`, I `require` my other files. In my `index.js` I create a `net` server before requiring my other files. I found that by doing that, I could just use plain `http` in my other files. Example:

``` javascript
// index.js
'use strict';
const runtime = require('runtimejs');
const net = require('net');
const wrapper = net.createServer(function(socket) {
  socket.once('data', function(data) {
    socket.end();
    wrapper.close();
  });
});
wrapper.listen(0, function() {
  const socket = net.createConnection(wrapper.address().port);
  socket.write('hey');
  socket.end();
});

require('other.js');
```

``` javascript
// other.js
const runtime = require('runtimejs');
const http = require('http');
http.get('http://www.google.com', function(res) {
  // prints '200'
  runtime.tty.print(res.statusCode);
});
```

So maybe in `runtime-node-net`'s `index.js` you could create a server and close it before exporting the module, i think it would be a solution that would allow node compatability. Should I open a PR?
EDIT: Just tested it on `runtime-node-net`'s `index.js` and removed it from my `index.js` and it works. Opening a PR...

**Comment by kesla** [comment]
let's fix the real problem about this in #3 instead. This should not be necessary.

**Comment by facekapow** [comment]
OK, closing Pull request.
